themes/digital.gov/src/scss/new/_authors-list.scss

```diff
@@ -2,53 +2,82 @@
 // NOTE: The authors-list is included twice on the page. Once for desktop, the other for mobile.
 @use "uswds-core" as *;
 
-.authors-list {
-  @include u-margin-top(0);
-  @include u-margin-bottom(7);
+// authors landing override
+.authors {
+  .authors-list .author {
+    border-top: 1px color("gray-10") solid;
+  }
 
-  .author {
-    @include u-margin-bottom("105");
-    @include u-padding-top("105");
-    @include u-border-top("1px", "solid", "gray-10");
-    @include u-flex("no-wrap");
+  .author__name {
+    @include u-font("sans", "xs");
+    @include u-line-height("sans", 1);
+    @include u-margin-y("05");
+    @include u-text("bold", "black", "no-underline");
 
-    img {
-      @include u-border("1px", "solid", "indigo-30");
+    &:hover {
+      @include u-border-bottom("2px", "accent-warm", "solid");
     }
+  }
```

Comment:
Added styles for overriding component styles when on [/authors](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-refactor-author-templates/authors/)